eng/Packages.props

```diff
@@ -99,7 +99,7 @@
     <PackageReference Update="NuGet.Packaging" Version="4.8.0" />
     <PackageReference Update="NUnit" Version="3.12.0" />  
     <PackageReference Update="NUnit3TestAdapter" Version="3.16.1" />
-    <PackageReference Update="Octokit" Version="0.40.0" />
+    <PackageReference Update="Octokit" Version="0.49.0" />
```

Comment:
The latest version available in dotnet-public is only 0.48.0.

Reply:
This one is on nuget and should be on dotnet-public by now.

Reply:
[It isn't](https://dev.azure.com/dnceng/public/_packaging?_a=package&feed=dotnet-public&package=Octokit&protocolType=NuGet&version=0.48.0&view=versions) and neither is 0.50.0, which is also on nuget now. I'm not sure how the sync is supposed to work; could something be wrong there?